src/molecules/Tabs/Tabs.native.js

```diff
@@ -76,10 +76,12 @@ const Tabs = ({ children, defaultValue, value, onChange }) => {
 
   const onIndexChange = useCallback(
     (newIndex) => {
-      if (!isDefined(value)) {
-        setIndex(newIndex);
+      if (typeof newIndex === 'number') {
+        if (!isDefined(value)) {
```

Comment:
Use `isEmpty`.

Reply:
Won't it be better if I do a check for number since I _want_ a number?

Reply:
You can keep that, I'm asking to use `isEmpty` instead of `isDefined`.

Reply:
IsDefined and isEmpty are different